MapboxAndroidDemo/src/gpservices/java/com/mapbox/mapboxandroiddemo/MapboxApplication.java

```diff
@@ -15,7 +15,14 @@ public void onCreate() {
     super.onCreate();
     initializeFirebaseApp();
     setUpPicasso();
-    Mapbox.getInstance(this, getString(R.string.access_token));
+    if (BuildConfig.FLAVOR.matches("gpsservices")) {
```

Comment:
I think that we'd want to have different folders for the two flavors (and include two versions of this file), instead of using conditional checks, under `src/nogpservices/java/com/mapbox/mapboxandroiddemo/MapboxApplication.java`.